### PR TITLE
Added NGINX Unit to the Servers list

### PR DESCRIPTION
**Pull request by artemkonev**
Added NGINX Unit (https://unit.nginx.org) to the list of servers supporting the ASGI specification.

**Checklist**

- [x] This project is explicitly related to ASGI.
- [x] The new list entry contains a project name, URL and description.

**What is this project?**

[NGINX Unit](https://unit.nginx.org) - A polyglot app server that supports ASGI 3.0 for Python.

**Do you know about other similar projects?**

Yes.

**If so, how is this one different?**

Perhaps, the most notable difference is that Unit is not a Python-only app server. It's mostly written in C, dynamically configurable, and functions as a static server/proxy as well.

---

_Anyone who agrees with this pull request can add a 👍._

### CODE REVIEW

**Comment by tippexs** [comment]
Hi @florimondmanca 
Just came across this PR again and wanted to ask if there is anything else you need from us? Would love to see NGINX Unit in the servers list :) Let me know if there is anything else we can do to make this happen.

**Review by tippexs — CHANGES_REQUESTED** [review]
It should say `A universal web app server that supports ASGI 3.0 for Python.`

File: README.md [hunk 1]
```diff
@@ -202,6 +202,7 @@ _Web servers for ASGI applications._
 
 - [Daphne](http://github.com/django/daphne) - An HTTP, HTTP2 and WebSocket protocol server for ASGI, developed to power Django Channels.
 - [Hypercorn](https://pgjones.gitlab.io/hypercorn/index.html) - An ASGI server based on the sans-io hyper, h11, h2, and wsproto libraries. Supports HTTP/1, HTTP/2, WebSockets, ASGI 2.0 and ASGI 3.0. Compatible with asyncio, uvloop and trio worker types.
+- [NGINX Unit](https://unit.nginx.org) - A polyglot app server that supports ASGI 3.0 for Python.
```
Comment (florimondmanca) [inline README.md:205]:
```suggestion
- [NGINX Unit](https://unit.nginx.org) - A universal web app server that supports ASGI.
```

**Review by florimondmanca — APPROVED** [review]

**Comment by florimondmanca** [comment]
@tippexs Thanks for the heads up, I think this is ready to merge!

**Pull request merged**